utils/webcomponents-utils.js

```diff
@@ -0,0 +1,33 @@
+(function() {
```

Comment:
Add some comment to know where did you get this file from to be able to update it in the future.

Reply:
I don't know where this script comes from. I looked for the origins but I didn't found anything :/
It's frequently used when working with Web Components.